### PR TITLE
Updated property update to have selects for region and districtg

### PR DESCRIPTION
**Pull request by FuriousLlama**
image

### CODE REVIEW

**Review by stairaku — APPROVED** [review]

File: backend/api/Areas/Property/Models/Property/PropertyModel.cs [hunk 1]
```diff
@@ -112,6 +112,16 @@ public class PropertyModel : BaseModel
...
+        /// <summary>
+        /// get/set - District type.
+        /// </summary>
+        public TypeModel<short> DistrictType { get; set; }
+
+        /// <summary>
+        /// get/set - Region type.
+        /// </summary>
+        public TypeModel<short> RegionType { get; set; }
+
```
Comment (asanchezr) [inline backend/api/Areas/Property/Models/Property/PropertyModel.cs:124]: nit: would prefer if these are named District, Region to align better with the "concept" Property. Ideally in the future we would want to refactor this specific model into the concept one

File: frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx [hunk 2]
```diff
@@ -72,13 +72,16 @@ const FormComponent: React.FC<FormikProps<IPropertyDetailsForm>> = ({ values })
...
         <SectionField label="MOTI region">
-          <Text field="motiRegion.REGION_NAME" />
+          <Text field="regionType.description" />
         </SectionField>
         <SectionField label="Highways district">
           <InlineContainer>
-            <Text field="highwaysDistrict.DISTRICT_NUMBER" />
-            {'-'}
-            <Text field="highwaysDistrict.DISTRICT_NAME" />
+            {values.districtType?.description !== 'Cannot determine' && (
```
Comment (asanchezr) [inline frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx:79]: nit: would you consider using a constant for 'Cannot determine' ? might help re-use in other places

File: frontend/src/interfaces/IPropertyApiModel.ts [hunk 3]
```diff
@@ -36,6 +36,9 @@ export interface IPropertyApiModel extends Api_ConcurrentVersion {
   propertyType?: Api_TypeCode<string>;
   status?: Api_TypeCode<string>;
 
+  regionType?: Api_TypeCode<number>;
+  districtType?: Api_TypeCode<number>;
+
```
Comment (asanchezr) [inline frontend/src/interfaces/IPropertyApiModel.ts:41]: as per previous comment would love to see this named just region, district to align with concept API_Property

**Review by asanchezr — APPROVED** [review]

File: frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx (hunk 2 above)
Comment (FuriousLlama) [reply frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx:79]: I thought about but given it only is needed on this page decided to leave it as is. We could have a enum for regions in the future

File: frontend/src/interfaces/IPropertyApiModel.ts (hunk 3 above)
Comment (FuriousLlama) [reply frontend/src/interfaces/IPropertyApiModel.ts:41]: the problem is that these are not part of the concept, but the old model. We should get rid of it completly

**Pull request merged**